app/views/foods/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="w-full max-w-md">
+  <h3 class="h3">Add New Food</h3>
+  <%= form_with model: @food, class: "form", scope: :foods do |f| %>
+
+    <div class="mb-3">
+    <%= f.label :Name, class: "form-label" %>
+    <%= f.text_field :name, placeholder: "Add Name of Food", required: true, class: "form-control" %>
+    </div>
+
+    <div class="mb-3">
+    <%= f.label :Measurement_Unit, class: "form-label" %>
+    <%= f.text_field :measurement_unit, placeholder: "Enter Measurement Unit", required: true, class: "form-control" %>
+    </div>
+    
+    <div class="mb-3">
+    <%= f.label :Unit_Price, class: "form-label" %>
+    <%= f.text_field :price, placeholder: "Unit Price", required: true, class: "form-control" %>
+    </div>
+
+    <div class="mb-3">
+    <%= f.label :Quantity, class: "form-label" %>
+    <%= f.text_field :quantity, placeholder: "Quantity", required: true, class: "form-control" %>
+    </div>
+
+    <%= f.submit "Add Food", class: "btn btn-primary" %>
+  <% end %>
```

Comment:
- [ **OPTIONAL** ] For the DRY rule and best practices, I think it would be better to use **partial files** whenever possible in your different views. I suggest you refactor your views by doing so, **[here](https://dev.to/isalevine/let-s-use-rails-partial-views-to-render-art-from-magic-the-gathering-cards-5d8l)** is a good article about partials. But good job with your project :+1: :+1: :100: